trampoline/src/main/java/org/ernest/applications/trampoline/controller/SettingsController.java

```diff
@@ -126,4 +127,16 @@ public void checkoutAndPullAndRestart(@RequestParam(value="id") String id, @Requ
 		List<String> intancesIds = ecosystemManager.getEcosystem().getInstances().stream().filter(i-> i.getMicroserviceId().equals(id)).map(Instance::getId).collect(Collectors.toList());
 		intancesIds.forEach(intancesId-> ecosystemManager.restartInstance(intancesId));
 	}
+
+	@RequestMapping(value= "/git/save", method = RequestMethod.POST)
+	@ResponseBody
+	public void saveGitCred(@RequestParam(value="user") String user,@RequestParam(value="pass") String pass) {
+		gitManager.saveCred(user, pass);
+	}
+
+	@RequestMapping(value= "/git/clean", method = RequestMethod.GET)
+	@ResponseBody
+	public void cleanGitCred() {
+		gitManager.cleanCred();
+	}
 }
```

Comment:
newline

Reply:
I don't know what happen with these newlines... I will try to avoid it from now on... but I think github should solve it :)

Reply:
Haha, yeah, they do such a good job making it look bad, but won't clean it up.